pkg/util/http.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import (
+	"strings"
+)
+
+// IsRequestBodyTooLarge returns true if the error is "http: request body too large".
+func IsRequestBodyTooLarge(err error) bool {
```

Comment:
does this belong in /pkg/api?

honestly it only seems to bei n use in `/modules/frontend`. should we just put it there?

Reply:
makes sense. moved to `frontend/handler.go`